src/test/lambda/local/util.ts

```diff
@@ -7,8 +7,25 @@
 
 import * as os from 'os'
 import * as path from 'path'
+import { Uri, WorkspaceFolder } from 'vscode'
 import * as filesystem from '../../../shared/filesystem'
 
+export async function createWorkspaceFolder(prefix: string): Promise<{
```

Comment:
`prefix` isn't used in the method

Reply:
Did you intend to pass it to `createTemporaryDirectory`?

Reply:
Yep, good catch.